Formula/scdoc.rb

```diff
@@ -0,0 +1,18 @@
+class Scdoc < Formula
+  desc "Tool designed to make the process of writing man pages more friendly"
+  homepage "https://git.sr.ht/~sircmpwn/scdoc/"
+  url "https://git.sr.ht/~sircmpwn/scdoc/snapshot/scdoc-1.4.1.tar.xz"
+  sha256 "2bc96daae0ddfb858305e550eb00a544a7a6255edd0bd169dcb2fc0c035eca92"
+
+  def install
+    system "make", "LDFLAGS=", "all"
+    bin.install "scdoc"
```

Comment:
Is there not a `make install` we can use?

Reply:
There is but it uses the `install` command and some flags that macOS' built-in `install` doesn't support. I tried doing some search/replace stuff on the Makefile but it grew to be a huge mess. `bin.install` and `man1.install` just seemed cleaner.

Reply:
Oh I assume they're using GNU extensions. It's probably something fairly simple to fix; usually you can support the BSD `install` fairly easily, you just end up having to take a few more steps to reach the same destination as you would with GNU `install`. I'll take a look.